templates/app/videos.phtml

```diff
@@ -13,17 +13,26 @@ $this->headTitle($title);
 <div class="wrapper content">
     <h1><?php echo $title; ?></h1>
```

Comment:
Technically this should be escaped, though I can see it doesn't need to be in this case.